app/helpers/layout_helper.rb

```diff
@@ -63,6 +63,17 @@ def title(page_title, page_header = nil)
     @page_header ||= page_header || page_title.to_s
   end
 
+  def edit_title(obj, page_header: nil, attribute: :to_s)
+    valid_obj = obj
+    unless obj.valid?
+      # we need to override changed values with previous ones,
+      # in order to show the valid name in the page title after an unsuccessful editing
+      old_attributes = obj.attributes.merge(obj.changed_attributes)
+      valid_obj = obj.class.new(old_attributes)
```

Comment:
you could simplify this and remove the merge using `obj.class.new(attributes_in_database)`